### PR TITLE
add the IronCheckedElementBehaviour

### PR DESCRIPTION
**Pull request by notwaldorf**
Also added the `a11ySuite` for testing.

:pager: @cdata

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
     "polymer": "Polymer/polymer#^1.1.0",
     "paper-ripple": "PolymerElements/paper-ripple#^1.0.0",
     "paper-styles": "PolymerElements/paper-styles#^1.0.0",
-    "paper-behaviors": "PolymerElements/paper-behaviors#^1.0.0"
+    "paper-behaviors": "PolymerElements/paper-behaviors#^1.0.0",
+    "iron-checked-element-behavior": "PolymerElements/iron-checked-element-behavior#^1.0.0"
```
Comment (cdata) [inline bower.json:27]: :gb:

File: paper-toggle-button.html [hunk 2]
```diff
@@ -13,6 +13,7 @@
 <link rel="import" href="../paper-styles/default-theme.html">
 <link rel="import" href="../paper-ripple/paper-ripple.html">
 <link rel="import" href="../paper-behaviors/paper-inky-focus-behavior.html">
+<link rel="import" href="../iron-checked-element-behavior/iron-checked-element-behavior.html">
```
Comment (cdata) [inline paper-toggle-button.html:16]: :gb:

File: paper-toggle-button.html [hunk 3]
```diff
@@ -168,7 +169,8 @@
       is: 'paper-toggle-button',
 
       behaviors: [
-        Polymer.PaperInkyFocusBehavior
+        Polymer.PaperInkyFocusBehavior,
+        Polymer.IronCheckedElementBehavior
```
Comment (cdata) [inline paper-toggle-button.html:173]: :gb:

File: test/basic.html [hunk 4]
```diff
@@ -80,6 +82,8 @@
           console.log(b1.getAttribute('role'));
           assert.isTrue(b1.getAttribute('role') == 'button');
         });
+
+        a11ySuite('Basic');
```
Comment (cdata) [inline test/basic.html:86]: :gb: :gb: :gb: :gb: :gb: :gb: :gb: :gb: :gb: :gb: :gb: :gb: :gb: :gb:

**Comment by cdata** [comment]
:gb: :trumpet: :trumpet: :trumpet: :gb:

:maple_leaf: :goat: :maple_leaf: 

LGTM :100:

**Comment by samccone** [comment]
I approve this use of emoji :star:

**Pull request merged**